waterdynamics/tests/test_waterdynamics.py

```diff
@@ -43,15 +43,29 @@ def universe():
 
 def test_WaterOrientationalRelaxation(universe):
     wor = waterdynamics.WaterOrientationalRelaxation(
-        universe, SELECTION1, 0, 5, 2)
+        universe, SELECTION1, 0, 5, 2, order=2)
     wor.run()
     assert_almost_equal(wor.timeseries[1][2], 0.35887,
                         decimal=5)
 
 
 def test_WaterOrientationalRelaxation_zeroMolecules(universe):
     wor_zero = waterdynamics.WaterOrientationalRelaxation(
-        universe, SELECTION2, 0, 5, 2)
+        universe, SELECTION2, 0, 5, 2, order=2)
+    wor_zero.run()
+    assert_almost_equal(wor_zero.timeseries[1], (0.0, 0.0, 0.0))
+
+def test_WaterOrientationalRelaxation(universe):
+    wor = waterdynamics.WaterOrientationalRelaxation(
+        universe, SELECTION1, 0, 5, 2, order=1)
+    wor.run()
+    assert_almost_equal(wor.timeseries[1][2], 0.71486,
+                        decimal=5)
+
+
+def test_WaterOrientationalRelaxation_zeroMolecules(universe):
```

Comment:
rename so that each test is unique
```suggestion
def test_WaterOrientationalRelaxation_order_1_zeroMolecules(universe):
```